k8s/base/observability.yaml

```
@@ -0,0 +1,254 @@
+    matchLabels:
+      app: otel-collector
+  template:
+    metadata:
+      labels:
+        app: otel-collector
+    spec:
+      containers:
+      - name: otel-collector
+        image: otel/opentelemetry-collector-contrib:0.90.1
+        args:
+          - --config=/etc/otel-collector-config/config.yaml
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)